profile-widget/profilewidget2.cpp

```diff
@@ -1066,7 +1068,8 @@ void ProfileWidget2::scrollViewTo(const QPoint &pos)
 void ProfileWidget2::mouseMoveEvent(QMouseEvent *event)
 {
 	QPointF pos = mapToScene(event->pos());
-	toolTipItem->refresh(pos);
+	toolTipItem->refresh(mapToScene(mapFromGlobal(QCursor::pos())));
```

Comment:
There is a lot of position magic going on. 

The `event->pos()` and the `mapToScene(mapFromGlobal(QCursor::pos()))` are almost the same. When moving the mouse slowly, I see on my system a (2, 2) pixel offset between the two. So when we would do the 2 refresh calls differently, a tiny jump of info box might occur. Obviously, we do not have an event to pull the pos() from in the plotDive function. 

When moving (very) fast, inside, and inside->outside with the mouse, the differences can be bigger (and even very big). This is also in the current master (without this PR, and likely forever, but nobody ever noticed). Try the following: move fast inside, sudden stop, and click a mouse button (without moving the mouse). Carefully watch the time in the info box. This click repaints the info fox. There is a very big chance that the time will change. In fact, the data in the box was wrong, and is now correct after the repaint. It is wrong in the sense that the mouse it at point A, but the data shows data from point B. Repaint causes data from point A to be drawn.

I found this behavior because it annoyed me, that selecting a different DC (causing a repaint) can change the time of the data in the info box (and all other values). But after realizing that it always has been this way, I sort of accepted this. It seems a fundamental problem that the `QCursor::pos())` and the `event->pos()` can be different when moving fast (just depending when al the handlers are called in the Qt's event loop).

Reply:
To obfuscate it a little more :-) The toolTip is drawn at position: `toolTipItem->pos()`. The data inside in it is drawn for the location of the mouse pointer which is controlled by `toolTipItem->refresh(<mouse position>)`. Basically, these are unrelated. Above, I was only talking about the latter, and Dirk's question was about the first.